### ISSUE TITLE
StrictMode Config

### ISSUE THREAD

**Issue by enri90** [vikejs/vike-react#109]
Do you think it makes sense for you to implement react StrictMode in the config? It would help me avoid situations

https://nextjs.org/docs/pages/api-reference/next-config-js/reactStrictMode

```
export default {
  Head,
  ssr: false,
  reactStrictMode: false,
  extends: [vikeReact, vikeReactQuery],
  passToClient: ['routeParams'],
} satisfies Config;
``` 

or

```
function Wrapper({ children }: { children: React.ReactNode }) {
  return (
    <React.StrictMode>
      {children}
    </React.StrictMode>
  )
}
``` 

or

```
export default defineConfig(async () => ({
  plugins: [
    react(),
    vike({ prerender: true ,reactStrictMode: false }),

``` 
Thx

**Comment by brillout** [comment]
Yes, `vike-react` should probably add the scrict wrapper and enable it by default. In the meantime an easy workaround is to use `<Wrapper>`.

**Comment by Nickhoyer** [comment]
Right now React.StrictMode is added by default in `onRenderHtml.tsx`.
React.StrictMode runs useEffects twice even with an empty array, which can cause bugs for some users.
Just want to give my vote that this gets fixed, I think it's fine that it's on by default, but should definitely be toggleable

**Comment by brillout** [comment]
I agree, I’ll release that later today/tomorrow.

**Comment by brillout** [comment]
Released in `0.4.9`.
